lib/cn/cn_tree_cursor.h

```diff
@@ -6,27 +6,50 @@
 #ifndef HSE_KVDB_CN_CN_TREE_CURSOR_H
 #define HSE_KVDB_CN_CN_TREE_CURSOR_H
 
-/* MTF_MOCK_DECL(cn_tree_cursor) */
-
 #include <hse_util/inttypes.h>
 #include <hse_util/hse_err.h>
+#include <hse_util/table.h>
+
+#include <hse_ikvdb/kvs.h>
+#include <hse_ikvdb/cursor.h>
+
+/* MTF_MOCK_DECL(cn_tree_cursor) */
 
 struct cn_cursor;
 struct cn_tree;
+struct cn_tree_node;
 struct kvs_ktuple;
 
-/* MTF_MOCK */
-merr_t
-cn_tree_cursor_create(struct cn_cursor *cur, struct cn_tree *tree);
+struct kvref {
+    struct kvset *kvset;
+};
```

Comment:
What's the value of `struct kvref`?  Why not just use `kvset *` ?  I'm gonna guess:  so it can be used easiliy in `struct table` entries?

Reply:
Right, and I just left it as it was. But I could remove this and use `struct kvset **` instead in the table.